doc/user_download_and_install_windows.rst.template

```diff
@@ -0,0 +1,355 @@
+
+In order to compile and test HORTON, you need to install relatively recent
+versions of the following programs/libraries:
+
+${dependencies_rst}
+
+
+Installing dependencies with a package manager
+----------------------------------------------
+
+With Ubuntu, most of these dependencies can be installed with a `apt-get`:
+
+* **Ubuntu Linux 16.04**:
+
+  .. code-block:: bash
+
+    sudo apt-get install ${dependencies_ubuntu_16_deb}
+
+  Note that Ubuntu 16.04 does not have up-to-date packages for all dependencies. LibInt2
+  must be compiled manually as explained in :ref:`linux_manual_dependency_install`. (The
+  beta version of LibInt2 that comes with Ubuntu 16.04 is compiled with a too low
+  LIBINT2_MAX_AM_ERI value.)
+
+* **Ubuntu Linux 15.04 and 14.04**:
+
+  .. code-block:: bash
+
+    sudo apt-get install ${dependencies_ubuntu_15_deb}
+    pip install --user --upgrade ${dependencies_ubuntu_15_pip}
+
+  Note that Ubuntu 15.04 and 14.04 do not have up-to-date packages for all dependencies.
+  Hence, some are installed with PIP. LibInt2 must be compiled manually as explained in
+  :ref:`linux_manual_dependency_install`.
+
+* **Precautionary Measures:
+  Those of you that are installing HORTON as a dependency for other possibly "top
+  secret" stuff (you know who you are) should also upgrade `scipy` and `matplotlib`.
+
+  Scipy is quite simple to upgrade:
+  .. code-block:: bash
+
+    pip install --user --upgrade scipy
+
+  Matplotlib requires a little more:
+  .. code-block:: bash
+
+    sudo apt-get install libfreetype6-dev libpng pkg-config
+    ln -s /usr/include/freetype2/ft2build.h /usr/include/
+    pip install --user --upgrade matplotlib
+
+
+.. _linux_manual_dependency_install:
+
+Installing dependencies manually
+--------------------------------
+
+If the package manager of your Linux distribution does not have the desired
+packages (or the right versions), you have to install them manually, e.g.
+download and execute an installer, or download and unpack a binary package. On
+HPC environments a compilation from scratch is recommended.
+
+**LibXC**
+
+The directory ``tooals/qa`` of the HORTON source tree contains scripts to download and
+build LibXC, which will work on most systems:
+
+.. code-block:: bash
+
+    ${custom_install_libxc}
+
+This results in a LibXC library suitable for compiling HORTON. If this fails,
+consult your local Linux guru to build LibXC. For more info about LibXC, check
+the website: http://www.tddft.org/programs/octopus/wiki/index.php/Libxc
+
+**LibInt2**
+
+The directory ``tooals/qa`` of the HORTON source tree contains scripts to download and
+build LibInt2, which will work on most systems:
+
+.. code-block:: bash
+
+    ${custom_install_libint}
+
+The compilation of LibInt2 takes a few minutes and results in a library suitable for
+compiling HORTON. If this fails, consult your local Linux guru to build LibInt2. For more
+info about LibInt2, check the website: http://sourceforge.net/p/libint/home
```

Comment:
Actually this is the website for libint I believe: https://github.com/evaleev/libint